### PR TITLE
Rebased: Allows field labels to be set in config

### PR DESCRIPTION
**Pull request by felix1m**
see #203 
credits to @maebert

### CODE REVIEW

**Comment by jonafato** [comment]
This failed to build for a few reasons:

- There are a few PEP8 formatting errors. These should be easy fixes.
- On Python 3, `unicode` is removed. This can be solved by conditionally using `unicode` or `str` depending version (e.g. `sys.version_info[0] > 3`).

**Comment by mattupstate** [comment]
You can use `flask_security.utils.string_types` and `flask_security.utils.text_type` instead of rolling your own check.

**Comment by jonafato** [comment]
@mattupstate +1 I knew that had to exist somewhere in the project but overlooked utils for some reason.

**Comment by felix1m** [comment]
someone still interested in getting this merged? Did not find the time so far..

**Comment by felix1m** [comment]
okay, i found the time, took only a second :)

**Comment by ArthurPBressan** [comment]
Hello, is there any plans to merge this? I could really use this.

File: flask_security/core.py [hunk 1]
```diff
@@ -175,6 +175,23 @@
         'Please reauthenticate to access this page.', 'info'),
 }
 
+
+_default_field_labels = {
+    'EMAIL': 'Email Address',
```
Comment (jirikuncar) [inline flask_security/core.py:180]: It would be nice to mark the values with `_(...)` for better i18n support.

**Pull request closed without merging**